meta-lxatac-software/recipes-devtools/python/python3-labgrid/configuration.yaml

```diff
@@ -103,3 +103,17 @@ lxatac-usb-ports-p{{idx}}.{{if}}:
       '@ID_PATH': '{{sysfs}}:1.{{if}}'
 {% endfor %}
 {% endfor %}
+
+
+## Interal TFTP Server
+tftp:
+  TFTPProvider:
+    internal: "/srv/tftp/"
+    external: "/"
+
+## Internal HTTP Server
+http:
+  HTTPProvider:
+    internal: "/srv/www/"
+    external: "/srv/"
```

Comment:
Should the `external` entry refer to an URL instead? That is how I read the [documentation](https://labgrid.readthedocs.io/en/latest/configuration.html#httpprovider).

Reply:
Ok, in this case the `HTTPProvider` example should be updated to use the `{{hostname}}`-template.